src/playermat/GenericCounter.ttslua

```diff
@@ -0,0 +1,39 @@
+MIN_VALUE = -99
```

Comment:
I think we can set this to zero.  We don't have anything that tracks negatives, right?

Reply:
probably not needed, but wouldn't hurt to let this in there just in case someone is using that for something?

Reply:
I can only think of one use case even at the fringe (someone who counts health/sanity down rather than up, in Dim Carcosa) but I know I see players in my group often overclicking and going negative when healing or paying for things.  I think we should floor it at zero, but it's a minor annoyance, can leave it for later.